src/main/java/org/zeromq/ZMQ.java

```diff
@@ -336,7 +338,7 @@ public static boolean proxy(Socket frontend, Socket backend, Socket capture, Soc
                              frontend.base,
                              backend.base,
                              capture == null ? null : capture.base,
-                             control == null ? null : control.base);
+                                             control == null ? null : control.base);
```

Comment:
This indentation looks strange -- I think the previous version is more correct, as we're lining up the method arguments vertically.

Reply:
My bad, this should not be in the comit.